usm_rest_api/v1/urls.py

```diff
@@ -2,10 +2,16 @@
 from rest_framework import routers
 from .views import views as v1_views
 
+from djcelery import views as celery_views
+
 router = routers.DefaultRouter(trailing_slash=False)
 router.register(r'users', v1_views.UserViewSet)
 router.register(r'clusters', v1_views.ClusterViewSet)
```

Comment:
Although I missed this in the past, this endpoint should be name-spaced to live under /ceph or /gluster. What do you think?